gallery/src/PageHeader/PageHeader.scss

```diff
@@ -13,10 +16,13 @@
 
   &__home-link {
     align-items: center;
-    border-right: 1px solid var(--nx-swatch-grey-30);
+    border-right: 1px solid var(--nx-color-text);
     display: flex;
     padding-right: var(--nx-spacing-6x);
     margin-right: var(--nx-spacing-6x);
+    @include nx-dark-mode-helpers.dark-mode {
+      border-color: var(--nx-swatch-indigo-25);
```

Comment:
There's a `--nx-color-page-header-border` for this.

Reply:
> There's a `--nx-color-page-header-border` for this.

Thanks, updated here [ff40342](https://github.com/sonatype/sonatype-react-shared-components/pull/1025/commits/ff40342d4284c32c194dc8f8040d9737d8a796ee)